libfwupdplugin/fu-firmware-common.h

```diff
@@ -8,8 +8,39 @@
 
 #include <gio/gio.h>
 
+G_DEPRECATED_FOR(fu_firmware_strparse_uint4_safe)
 guint8		 fu_firmware_strparse_uint4		(const gchar	*data);
+G_DEPRECATED_FOR(fu_firmware_strparse_uint8_safe)
 guint8		 fu_firmware_strparse_uint8		(const gchar	*data);
+G_DEPRECATED_FOR(fu_firmware_strparse_uint16_safe)
 guint16		 fu_firmware_strparse_uint16		(const gchar	*data);
+G_DEPRECATED_FOR(fu_firmware_strparse_uint24_safe)
 guint32		 fu_firmware_strparse_uint24		(const gchar	*data);
+G_DEPRECATED_FOR(fu_firmware_strparse_uint32_safe)
```

Comment:
add this to the list of stuff to drop with ABI bump

Reply:
Fixed, thanks.